src/ply.rs

```diff
@@ -351,28 +365,25 @@ fn open(ply_file: &Path) -> Result<(BufReader<File>, i64, Vec<ReadingFn>)> {
 
     // We align the buffer of this 'BufReader' to points, so that we can index this buffer and know
     // that it will always contain full points to parse.
-    Ok((
-        BufReader::with_capacity(num_bytes_per_point * 1024, file),
-        header["vertex"].count,
+    Ok(PlyIterator {
+        reader: BufReader::with_capacity(num_bytes_per_point * 1024, file),
         readers,
-    ))
+        num_total_points: header["vertex"].count,
+        offset: header.offset,
+    })
 }
 
 /// Abstraction to read binary points from ply files into points.
 pub struct PlyIterator {
     reader: BufReader<File>,
     readers: Vec<ReadingFn>,
     pub num_total_points: i64,
+    offset: Vector3<f64>,
 }
 
 impl PlyIterator {
     pub fn from_file<P: AsRef<Path>>(ply_file: P) -> Result<Self> {
-        let (reader, num_total_points, readers) = open(ply_file.as_ref())?;
-        Ok(PlyIterator {
-            reader,
-            readers,
-            num_total_points,
-        })
+        open(ply_file.as_ref())
```

Comment:
Would it make sense to remove the `open` method and move it here, since `from_file` now is a one liner with (almost the) same signature?

Reply:
I agree.

Reply:
Done.